crates/symbolicator/src/utils/futures.rs

```diff
@@ -17,7 +13,7 @@ use crate::metrics::{self, prelude::*};
 /// You would mainly use this if you are dealing with a trait methods which deals with
 /// futures.  Trait methods can not be async/await and using this type in their return value
 /// allows to integrate with async await code.
-pub type BoxedFuture<T> = Pin<Box<dyn Future<Output = T>>>;
+pub type BoxedFuture<T> = Pin<Box<dyn Future<Output = T> + Send + 'static>>;
```

Comment:
I wonder if you can now use `futures::future::BoxFuture` for this instead of our own type alias.  (it's also in futures' prelude)